server/api/events/elfsight.ts

```diff
@@ -1,19 +1,14 @@
 import eventSourcesJSON from '@/assets/event_sources.json';
-import { logTimeElapsedSince, serverCacheMaxAgeSeconds, serverStaleWhileInvalidateSeconds, serverFetchHeaders } from '@/utils/util';
-import { url } from 'inspector';
+import { logTimeElapsedSince } from '@/utils/util';
 import { DateTime } from 'luxon';
 
-export default defineCachedEventHandler(async (event) => {
+export default defineEventHandler(async () => {
```

Comment:
We can't cache the event handler here, since that will overwrite the route rules